### PR TITLE
[T3A3][W11-A1]Tay Keming Justin

### PR DESCRIPTION
**Pull request by INCENDE**

### CODE REVIEW

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -7,53 +7,49 @@
...
+        street = new Street(split[1]);
+        unit = new Unit(split[2]);
+        postalCode = new PostalCode(split[3]);
     }
 
     @Override
     public String toString() {
-        return value;
+        return block.value + ", " + street.value + ", " + unit.value + ", " + postalCode.value;
     }
```
Comment (cmkumar87) [inline src/seedu/addressbook/data/person/Address.java]: consider abstracting the Delimiter, comma, with a variable name.

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -7,53 +7,49 @@
...
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = Block.EXAMPLE + ", " + Street.EXAMPLE + ", " + Unit.EXAMPLE + ", " + PostalCode.EXAMPLE;//"123, some street 80, #00-00, 123456";
+    public static final String ADDRESS_SPLIT_REGEX = ", ";
 
     private Block block;
     private Street street;
     private Unit unit;
     private PostalCode postalCode;
```
Comment (cmkumar87) [inline src/seedu/addressbook/data/person/Address.java]: How about accessors for these new instance variables?

**Comment by cmkumar87** [comment]
@INCENDE Good job, in general.

Please review the comments and close PR. We are now advised to follow this protocol for PR closing to ensure student read our comments.

**Comment by INCENDE** [comment]
thanks @cmkumar87

**Pull request closed without merging**